packages/cli/src/oclif/buildFlags.js

```diff
@@ -13,11 +13,21 @@ const baseFlags = {
     char: 'd',
     description: 'Show extra debugging output'
     // pull from env?
+  }),
+
+  // Indicates we're calling a command from another command so we know when not
+  // to print duplicate messages.
+  invokedFromAnotherCommand: flags.boolean({
+    hidden: true
   })
 };
 
 // didn't destruture these opts because I want them all on one object to be picked from
-const defaultOpts = { debug: true, format: false };
+const defaultOpts = {
+  debug: true,
+  format: false,
+  invokedFromAnotherCommand: true
```

Comment:
probably want this to default to `false`, right? most of the time, we're calling the commands from CLI (unless i'm getting the implementation reversed)

Reply:
Setting `true` here means all the commands have a hidden `--invokeFromAnotherCommand` flag by default. This flag is true only if you pass in `--invokeFromAnotherCommand` when running the command. Does that make sense?

Reply:
yep, this was totally me mixing up "defaults to being present on all commands" (👍) with "defaults to being true for all commands" (👎). all good here!